_pydev_bundle/pydev_is_thread_alive.py

```diff
@@ -4,7 +4,11 @@
 # circumstances).
 # It is required to debug threads started by start_new_thread in Python 3.4
 _temp = threading.Thread()
-if hasattr(_temp, "_is_stopped"):  # Python 3.x has this
+if hasattr(_temp, "_handle") and hasattr(_temp, "_started"):
```

Comment:
There's no longer an `_is_stopped` field in 3.13. I didn't want to use `is_alive` for the same reasons stated here. Some of the tests don't pass it with it either.

This logic is what is_alive does after it asserts.